Taggers/interface/CollectionDumper.h

```diff
@@ -31,7 +31,9 @@
 
 #include "flashgg/MicroAOD/interface/CutBasedClassifier.h"
 #include "flashgg/MicroAOD/interface/ClassNameClassifier.h"
+#include "flashgg/MicroAOD/interface/StageOneNameClassifier.h"
```

Comment:
Looking more into the details I am wondering if we need the HTXS splitting to be hardcoded into the CollectionDumper. Why can't we use the CutBasedDumper using a cut on the HTXS bin number to define the categories?

Reply:
it's a good point, and I can't recall the reason Seth preferred to do it this way originally. Perhaps we can discuss in person whether it's worth restructuring this or not.